agent/engine/ordering_integ_test.go

```diff
@@ -31,6 +32,12 @@ const orderingTimeout = 90 * time.Second
 // TestDependencyHealthCheck is a happy-case integration test that considers a workflow with a HEALTHY dependency
 // condition. We ensure that the task can be both started and stopped.
 func TestDependencyHealthCheck(t *testing.T) {
+	// Skip these tests on WS 2016 until the failures are root-caused.
+	isWindows2016, err := config.IsWindows2016()
```

Comment:
invoking ```IsWindows2016()``` inside each test means we execute the ```exec.Cmd(powershell cmd)``` N numbers of times? (N - no.of tests in this file) right?

wouldn't it be better to invoke it once and store the result?

Reply:
We could optimize this but the plan is to not skip these tests for a long period of time. We should remove this as soon as possible.